.env.example

```diff
@@ -9,3 +9,11 @@
 
 # DIRECT_URL is given to you if using Neon. If using another postgres instance, set it to the same value as DATABASE_URL
 # DIRECT_URL=postgres://postgres:123456@127.0.0.1:5432/dummy
+
+# NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY=pk_test_c2FmZS1yYXZlbi02OC5jbGVyay5hY2NvdW50cy5kZXYk
```

Comment:
is this the publishable key for the new CFD organization? This is fine to un-comment

Reply:
Yeah this can be uncommented